### PR TITLE
[development IF-9] user clicks on announcement it will redirect to announcement forum page

### PR DESCRIPTION
**Pull request by KJsharmila**

### CODE REVIEW

**Comment by asimha** [comment]
Please remove the unwanted files

File: app/assets/javascripts/announcements.js.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (kedarnag138) [inline app/assets/javascripts/announcements.js.coffee]: Remove this file.

File: app/assets/stylesheets/announcements.css.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the announcements controller here.
```
Comment (kedarnag138) [inline app/assets/stylesheets/announcements.css.scss]: Remove this file.

File: app/helpers/announcements_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module AnnouncementsHelper
```
Comment (kedarnag138) [inline app/helpers/announcements_helper.rb]: Remove this file.

File: app/controllers/announcements_controller.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
+class AnnouncementsController < ApplicationController
+	def index
+	end
+	def create
+		@announcement=Announcement.new(announcement_params)
```
Comment (asimha) [inline app/controllers/announcements_controller.rb:5]: Is This thing completed?

**Pull request merged**